syntax_and_semantics/case.md

```diff
@@ -71,6 +71,16 @@ elsif tmp.odd?
 end
 ```
 
+You may use `then` after the `when` condition. This is most frequently used to place the body of the `when` on a single line.
+
+```crystal
+case a
+when 1, 2 then puts "a is one or two"
```

Comment:
I'd rather you demonstrated returning values from the when body instead of causing side effects (printing). It's a much more common use of then.